tools/parse_module_type.py

```diff
@@ -224,6 +252,15 @@ def prequisitesCheck():
         return False
     return True
 
+    f = Path('./modulbeschriebe_ai.html')
+    if not f.is_file():
+        print('ERROR: file \'./modulbeschriebe_ai.html\' does not exist.')
+        print('To get started download the html file from \
+https://mycampus.hslu.ch/de-ch/info-i/dokumente-fuers-studium/bachelor/moduleinschreibung/modulbeschriebe/bachelor-artificial-intelligence-machine-learning/ \
+and save it as \'tools/modulbeschriebe_wi.html\'.')
```

Comment:
modulbeschriebe_wi.html should be modulbeschriebe_ai.html

Reply:
Duh... 🤦🏻‍♂️